utils/misc.ml

```diff
@@ -17,10 +17,13 @@
 
 exception Fatal_error
 
-let fatal_error msg =
-  prerr_string ">> Fatal error: "; prerr_endline msg; raise Fatal_error
+let fatal_errorf fmt =
+  Format.kfprintf
+    (fun _ -> raise Fatal_error)
+    Format.err_formatter
+    ("@?>> Fatal error: " ^^ fmt ^^ "@.")
```

Comment:
Should I use a box here?

Reply:
There may be some value to manifest the implicit `b` box, but this seems quite theoretical since the current error messages do not use any formatting hints (except `@.` that breaks all boxes anyway) .